src/05-derivatives/02-common-data-types.md

```diff
@@ -11,24 +11,23 @@ Template:
         <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
 ```
 
-Processing in this context means transformations of data that does not change
+Preprocessing in this context means transformations of data that do not change
 the number of dimensions of the input and are not explicitly covered by other
 data types in the specification. Examples:
 
- -  Motion-corrected, temporally denoised, and transformed to MNI space bold files.
- -  Inhomogeneity corrected and skull stripped T1w files.
- -  Motion-corrected DWI files.
+ -  Motion-corrected, temporally denoised, and transformed to MNI space BOLD series
+ -  Inhomogeneity corrected and skull stripped T1w files
+ -  Motion-corrected DWI files
  -  Time-domain filtered and ICA cleaned EEG data
```

Comment:
Please split "Time-domain filtering for EEG data" from "Spatial filtering, e.g. ICA cleaned EEG data" in the example, since these deal with different dimensions.

Reply:
the application of Maxfilter on MEG data could also be considered an example of preprocessing in which the dimensions don't change. Maxfilter is mentioned in the main spec [here](https://bids-specification.readthedocs.io/en/stable/99-appendices/06-meg-file-formats.html#neuromagelektamegin) and specifies the `proc` keyword (e.g. `proc-sss`) to be used in the derivatives.

Reply:
`proc-sss`... This is unfortunate. I don't really know what to do with derivatives specified prior to the derivatives spec.